index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="kr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/0f08ea7d4a.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <a href="/" class="nav-logo">
+          <img src="assets/logos/nav-logo.png" alt="img-nav-logo" />
+        </a>
+        <div class="nav-spacer"></div>
+        <a href="/login" class="nav-login-button"> 로그인 </a>
```

Comment:
❗️ 수정요청

지금 단계에서 페이지 이동을 구현하시기 위해서는 확장자를 포함시켜주셔야 합니다.
관련 [이슈](https://github.com/tauri-apps/tauri/issues/2328)를 첨부해드리니 한번 확인해보시면 좋을 것 같습니다.

또한 로그인 글자에 공백이 있는데 가능하면 필수적이지 않은 공백은 제거하는 것이 좋습니다.
css 로 스타일링 할때 의도치 않은 동작을 할 수도 있고, 의도가 있는 공백인지 아닌지 이해하기 어렵기 때문입니당.

```suggestion
        <a href="/login.html" class="nav-login-button">로그인</a>
```

Reply:
💬 여담
구현을 완료하신 후에는 작업하신 사항들이 잘 구현되엇는지, a tag, button tag들을 눌러보시면서 확인하시면 좋습니다~